### PR TITLE
Enhance blocked_by to skip specific platform/platforms

### PR DESCRIPTION
**Pull request by IonutBudeanu**

### CODE REVIEW

File: iris/tests/awesomebar/alt_up_down_keys_in_awesomebar.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def __init__(self):
         self.test_case_id = '108278'
         self.test_suite_id = '1902'
         self.exclude = Platform.LINUX
```
Comment (silviuFlorin) [inline iris/tests/awesomebar/alt_up_down_keys_in_awesomebar.py]: we can remove self.exclude if the blocked by field has the same platform

File: iris/tests/find_toolbar/navigate_quick_find.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ def __init__(self):
         self.test_case_id = '127260'
         self.test_suite_id = '2085'
         self.locales = ['en-US']
-        self.blocked_by = 'issue 1628'
```
Comment (silviuFlorin) [inline iris/tests/find_toolbar/navigate_quick_find.py]: We should leave 'issue_' as part of the test case ID to differentiate between Bugzilla and GitHub issues

File: iris/tests/awesomebar/alt_up_down_keys_in_awesomebar.py (hunk 1 above)
Comment (IonutBudeanu) [reply iris/tests/awesomebar/alt_up_down_keys_in_awesomebar.py]: Unnecessary self.exclude flags have been removed.

Thank You

File: iris/tests/find_toolbar/navigate_quick_find.py (hunk 2 above)
Comment (IonutBudeanu) [reply iris/tests/find_toolbar/navigate_quick_find.py]: GitHub issues have been updated in order to differentiate them from the Bugzilla issues.

Thanks.

**Pull request merged**